xbmc/guilib/GUIIncludes.cpp

```diff
@@ -530,6 +539,8 @@ const INFO::CSkinVariableString* CGUIIncludes::CreateSkinVariable(const std::str
 {
   std::map<std::string, TiXmlElement>::const_iterator it = m_skinvariables.find(name);
   if (it != m_skinvariables.end())
-    return INFO::CSkinVariable::CreateFromXML(it->second, context);
+    return INFO::CSkinVariable::CreateFromXML(it->second, context, [&](const std::string &expression) -> std::string {
```

Comment:
no need to capture any variable here so you can use `[]` I would think?

wouldn't it be possible to pass ResolveExpressions directly instead of using a lamba?